android/src/main/java/yeniellandestoy/native_shared_preferences/MethodCallHandlerImpl.java

```diff
@@ -210,6 +213,10 @@ private Map<String, Object> getAllPrefsFromDictionaries(List<String> keys) throw
   private Map<String, Object> parsePreferences(Map<String, ?> allPrefs) throws IOException {
     Map<String, Object> filteredPrefs = new HashMap<>();
     for (String key : allPrefs.keySet()) {
+      if (!allPrefs.containsKey(key)) {
```

Comment:
that's useless, you getting key from allPrefs so it's contained 100% of the time :)